ceph.spec.in

```diff
@@ -198,8 +195,8 @@
 %enable_devtoolset13
 %endif
 %else
-%if 0%{?enable_devtoolset11:1}
-%enable_devtoolset11
+%if 0%{?enable_devtoolset12:1}
+%enable_devtoolset12
```

Comment:
where do we install gts-12? if we can assume gcc >=12 is available, we could drop this line.

Reply:
okay. assuming we have the corresponding macro installed, we should have it.

Reply:
i think we'd expect install-deps.sh or ceph-build to install this where necessary

without ceph-build changes, i think it'll be hard to actually test this in ci

Reply:
@tchaikov can you help me understand what installs gts-13 for https://github.com/ceph/ceph/pull/55886?

Reply:
nevermind; setting gts_version=12 above causes us to install `gcc-toolset-%{gts_version}-build` which apparently makes this `%enable_devtoolset12` thing available

i see that working for the rpm builds in https://shaman.ceph.com/builds/ceph/wip-gcc-12/1b854c2b0aaaebb77f92763f72bc7dff83166319/